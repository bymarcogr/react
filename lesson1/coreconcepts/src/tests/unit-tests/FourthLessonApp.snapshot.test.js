import renderer from "react-test-renderer";
import FourthLessonApp from "../../components/fourthLessonApp";

it("renders FourthLessonApp correctly", () => {
  const tree = renderer.create(<FourthLessonApp />).toJSON();
  expect(tree).toMatchSnapshot();
});
