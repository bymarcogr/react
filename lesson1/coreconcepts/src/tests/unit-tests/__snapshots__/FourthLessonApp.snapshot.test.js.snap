// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FourthLessonApp correctly 1`] = `
<div>
  <h1
    className="text-success"
  >
    Fourth Lesson
  </h1>
  <div
    className="container-fluid"
  >
    <button
      className="btn btn-primary"
      onClick={[Function]}
      title="btn-show-dialog"
      type="button"
    >
      Show Generic Dialog
    </button>
    <br />
    <br />
    <button
      className="btn btn-danger"
      onClick={[Function]}
      title="btn-add-movie"
      type="button"
    >
      Add Movie
    </button>
  </div>
</div>
`;
